### PR TITLE
Miscellaneous doc updates pulled from other closed PRs

### PR DESCRIPTION
**Pull request by rogerluan**
This PR replaces https://github.com/fastlane/docs/pull/876, a stale PR that lacks CLA signing.
And it also replaces https://github.com/fastlane/docs/pull/788, a stale PR that got its original repository deleted so we can't fix the merge conflicts that originated in it.

# Google Credentials

Go to https://play.google.com/apps/publish/ and follow the screens alongside the google_credentials.md tutorial:

image

image

image

image

image

image

image

image

### CODE REVIEW

File: docs/includes/google-credentials.md [hunk 1]
```diff
@@ -2,12 +2,12 @@ Tip: If you see Google Play Console or Google Developer Console in your local la
...
 1. Open the [Google Play Console](https://play.google.com/apps/publish/)
 1. Click the **Settings** menu entry, followed by **API access**
-1. Click the **CREATE SERVICE ACCOUNT** button 
+1. Click the **CREATE SERVICE ACCOUNT** button
 1. Follow the **Google Developers Console** link in the dialog, which opens a new tab/window:
    1. Click the **CREATE SERVICE ACCOUNT** button at the top of the **Google Developers Console**
    1. Provide a `Service account name`
    1. Click **Select a role** and choose **Service Accounts > Service Account User**
-   1. Check the **Furnish a new private key** checkbox
+   1. Click the **Create key** button
```
Comment (janpio) [inline docs/includes/google-credentials.md:10]: Did you double check if this is really how it works? 
I think I had trouble confirming the PR that was coming in back then.

Comment (rogerluan) [reply docs/includes/google-credentials.md:10]: Very good point - I hadn't 🙈 
Just did, though, and made several adjustments. Also edited the PR description 🙏 

Thanks for pointing this out! I was a fool! 🙇

Comment (janpio) [reply docs/includes/google-credentials.md]: Don't worry, this UI is effectively why that other PR never got merged - it was just to hard to figure out what exactly was going on :D

File: docs/includes/google-credentials.md [hunk 2]
```diff
@@ -1,19 +1,22 @@
 Tip: If you see Google Play Console or Google Developer Console in your local language, add `&hl=en` at the end of the URL (before any `#...`) to switch to English.
 
-1. Open the [Google Play Console](https://play.google.com/apps/publish/)
-1. Click the **Settings** menu entry, followed by **API access**
-1. Click the **CREATE SERVICE ACCOUNT** button 
+
```
Comment (rogerluan) [inline docs/includes/google-credentials.md:3]:
```suggestion
```

**Pull request merged**